proto/tabletmanagerdata.proto

```diff
@@ -525,6 +525,7 @@ message VDiffReportOptions {
   bool only_pks = 1;
   bool debug_query = 2;
   string format = 3;
+  uint64 max_vdiff_report_rows = 4;
```

Comment:
Again, notice that the message is VDiffReportOptions so no need for vdiff in the var name.